src/components/Header.tsx

```diff
@@ -20,10 +23,11 @@ const Header = () => {
         <Border />
         <img src={Star} alt="sequencer status" />
         <SequencerStatus>
-          <span>Sequencer</span>
+          <span><Trans i18nKey="header.sequencer">Sequencer</Trans></span>
```

Comment:
I think most of the general words or simple translation can be translated using `t()` function rather than using `<Trans />` component.
We should use `<Trans />` component when the translation is spanning over multiple components.

so in this case you can just use `t('sequencer')`
What do you think?

FYI: https://react.i18next.com/latest/trans-component